lib/utilities/analytics-utils.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+var ANALYTICS_WHITELISTED_COMMANDS = [
```

Comment:
Is it possible to pull this list off of `this.commands`? We do that in `ember help` to get the list: https://github.com/ember-cli/ember-cli/blob/master/lib/commands/help.js but I don't know where it is populated.

I'd prefer not to hardcode our list of commands.